src/core/client/admin/components/UserStatus/BanModal.tsx

```diff
@@ -237,7 +262,12 @@ const BanModal: FunctionComponent<Props> = ({
                   {(viewerIsAdmin ||
                     viewerIsOrgAdmin ||
                     (viewerIsScoped && !viewerIsSingleSiteMod)) && (
-                    <Flex className={styles.sitesToggle} spacing={5}>
+                    <Flex
```

Comment:
What about just adding `isMultisite` to the conditional check that starts on line 262 for showing the All / Specific / No sites options? Then we wouldn't need to add these `hidden` styles.

Reply:
Fixed and incoming shortly